doc/source/guide.rst

```diff
@@ -120,6 +120,47 @@ on each line.
     branch, but ``gcov`` cannot discern which branch is executed.
 
 
+Out-of-Source Builds with CMake
+"""""""""""""""""""""""""""""""
```

Comment:
I think these should be all '~' as for the other headers.

Reply:
Changed.